cli/package.json

```diff
@@ -1 +1,35 @@
-{"version": "6.7.0"}
\ No newline at end of file
+{
+  "name": "protobufjs-cli",
+  "description": "Translates between file formats and generates static code as well as TypeScript definitions.",
+  "version": "7.0.0",
+  "author": "Daniel Wirtz <dcode+protobufjs@dcode.io>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dcodeIO/protobuf.js.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "bin": {
+    "pbjs": "bin/pbjs",
+    "pbts": "bin/pbts"
+  },
+  "scripts": {
+    "prepublish": "node scripts/prepublish"
+  },
+  "peerDependencies": {
+    "protobufjs": "^7.0.0"
```

Comment:
This doesn't make sense as a peer dependency. It is really a direct dependency: `pbjs` and `pbts` directly load `protobufjs`. As of recent versions of npm, [peer dependencies are not installed automatically](https://docs.npmjs.com/files/package.json#peerdependencies), so this will cause failures if people try to use `protobufjs-cli` as a standalone tool without also having their own dependency on `protobufjs`